packages/frontend-api/src/Resources/config/graphql-types/PriceDecorator.types.yml

```diff
@@ -0,0 +1,20 @@
+PriceDecorator:
+    type: object
+    config:
+        description: "Represents the price of the product"
+        fields:
+            id:
+                type: "ID!"
+                description: "Internal identifier"
+            priceWithVat:
+                type: "Money"
+                description: "Price with VAT"
+            priceWithoutVat:
+                type: "Money"
+                description: "Price without VAT"
+            vatAmount:
+                type: "Money"
+                description: "How much is going to build the new highway or agronomic subsidy"
+            isPriceFrom:
+                type: "Boolean!"
+                description: "Determines whether it's a final price or starting price"
```

Comment:
missing new empty line at the end

Reply:
Fixed https://github.com/shopsys/shopsys/pull/1471/commits/4baacbc382f1b3e5f6ba3028a53e2784b29d253f